R/render.R

```diff
@@ -140,7 +140,8 @@ render <- function(path = ".",
       type = type,
       config = config_contents,
       output_folder = output_folder,
-      path_language = main_language
+      path_language = main_language,
```

Comment:
this is starting to remind of pkgdown where we pass a pkg object to many, many functions. Maybe we could have a similar idea and create a book/website list with all this info so instead of passing main_language, path_language, etc, we'd just pass the list?

Reply:
(not in this PR! a thought for later)